src/views/prize-earned/prize-earned.scss

```diff
@@ -0,0 +1,46 @@
+h1 {
+  text-align: center;
+  margin-top: 72px;
+}
+
+.trophy-banner {
+  margin-top: 141px;
+  position: relative;
+}
+
+.trophy {
+  position: relative;
+  z-index: 1;
+  margin-top: -250px;
+  left: 50%;
+  margin-left: -187.5px; // half ot the image width
+}
+
+.top-line {
+  margin-top: -70px;
+}
+
+.parent {
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+.prize-details-container {
+  margin-bottom: 16px;
+  display: flex;
```

Comment:
I don't think you need `display: flex;` here since you're not setting anything on the contents (justify, align, grow/shrink/wrap, etc)

Reply:
This one I need to keep for centering purposes